admin-apis/updateApp.js

```diff
@@ -0,0 +1,63 @@
+import { generateApiSignature } from "./utils.js";
+import axios from "axios";
+import * as dotenv from "dotenv";
+import fetchAppListByAccountKey from "./fetchAllApps.js";
+
+dotenv.config();
+
+const updateAppUserWallet = async () => {
+  const colorCodes = ["#FFBF00", "#6495ED", "#CCCCFF"];
+  const updatedAppName = `App name - ${Math.floor(Math.random() * 10000)}`;
+  const updateUserWalletFields = {
+    displayName: `Display Name - ${Math.floor(Math.random() * 10000)}`,
+    button: {
+      backgroundColor:
+        colorCodes[Math.floor(Math.random() * colorCodes.length)],
+    },
+    theme: { style: "DARK" },
+    logoUrl: "https://google.com",
+  };
+
+  const initialAppList = await fetchAppListByAccountKey();
+  const initialApp = initialAppList[0];
```

Comment:
Here we are using first app in the list but for api_key changes we are using app_id from the env file. Let's be consistent across scripts.

Reply:
done